File: push/src/main/java/com/xiaomi/clientreport/processor/IWrite.java
package com.xiaomi.clientreport.processor;

import com.xiaomi.clientreport.data.BaseClientReport;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/clientreport/processor/IWrite.class */
public interface IWrite {
    void preProcess(BaseClientReport baseClientReport);

    void process();

    void write(BaseClientReport[] baseClientReportArr);
}

File: legacy/src/main/java/com/xiaomi/clientreport/processor/IWrite.kt
package com.xiaomi.clientreport.processor
import io.github.magisk317.mipush.protocol.model.*

import com.xiaomi.clientreport.data.BaseClientReport

interface IWrite {
    fun preProcess(baseClientReport: BaseClientReport)

    fun process()

    fun write(baseClientReportArr: Array<BaseClientReport>)
}
